src/catkin_pkg/package.py

```diff
@@ -696,7 +696,25 @@ def parse_package_string(data, filename=None, warnings=None):
         })
     nodes = [n for n in root.childNodes if n.nodeType == n.ELEMENT_NODE]
     unknown_tags = set([n.tagName for n in nodes if n.tagName not in known.keys()])
-    if unknown_tags:
+    if unknown_tags == set(['run_depend']) and pkg.package_format >= 2:
+        errors.append('\033[91mWARNING:')
```

Comment:
Please do not use ASCII codes for colorizing the output here.

Please also avoid "empty" `WARNING:` lines.